loop/storage/redis.js

```diff
@@ -3,7 +3,7 @@
  * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
 
 "use strict";
-var redis = require("redis");
+var redis = require("./redis_client");
```

Comment:
I actually consider this a feature since this allows us to tweak the client for all our use cases (for instance the default ports that changed, and we had to reflect this everywhere, which was a pain).